src/ssl.c

```diff
@@ -32871,7 +32869,7 @@ static int i2dProcessMembers(const void *src, byte *buf,
     return len;
 }
 
-int wolfSSL_ASN1_item_i2d(const void *src, byte **dest,
+int wolfSSL_ASN1_item_i2d(const void *src, byte **dest, // NOLINT(misc-no-recursion) /* fixme */
```

Comment:
What is the "fixme" here? Oh for the recursion? Can you try and fix it?

Reply:
fixing it seemed out of scope for this PR -- but I can take a look at it.

@SparkiDev fixed a recursion in sp in #4768 btw.

Reply:
OK I did indeed refactor this to eliminate the recursion.